module/vision/GoalDetector/src/GoalDetector.cpp

```diff
@@ -230,7 +230,7 @@ namespace module::vision {
                             g.measurements.back().position =
                                 cartesianToSpherical(Eigen::Vector3f(g.post.bottom * distance));
                             g.measurements.back().covariance =
-                                config.goal_angular_cov * Eigen::Vector3f(distance, 1, 1).asDiagonal();
+                                config.goal_angular_cov * Eigen::Vector3f(distance, 0.1, 0.1).asDiagonal();
```

Comment:
These definitely seem like they should be configuration values.
Also these things work better with inverse distance as the covariance then you can have a constant value in here